packages/aws-appsync/src/helpers/offline.ts

```diff
@@ -67,9 +67,20 @@ const getOpTypeFromOperationName = (opName = '') => {
     return result;
 };
 
+type CacheUpdateQuery = {
+    query: DocumentNode,
+    variables?: object,
+} | DocumentNode;
+
+type CacheUpdatesDefinitions = {
+    [key in CacheOperationTypes]?: CacheUpdateQuery | CacheUpdateQuery[]
+};
+
+type CacheUpdatesOptions = (variables?: object) => CacheUpdatesDefinitions | CacheUpdatesDefinitions;
+
 const buildSubscription = (
     subscriptionQuery: DocumentNode,
-    cacheUpdateQuery,
+    cacheUpdateQuery: DocumentNode,
```

Comment:
This shouldn't be CacheUpdateQuery type?

Reply:
Good point!